frontend/src/routers/AppRouter/AppRouter.tsx

```diff
@@ -211,6 +217,21 @@ export const AppRouter = () => {
 				<WithSessionSearchContext>
 					<WithErrorSearchContext>
 						<Routes>
+							<Route
+								path="/error-tags"
+								element={<ErrorTagsContainer />}
+							>
+								<Route index element={<ErrorTags />} />
+								<Route
+									path="/error-tags/admin"
```

Comment:
should this route be only accessible to us (highlight team)?

Reply:
Yeah. It should. Do we have an easy way to do this?

Reply:
Fixed!